uber/site_sections/summary.py

```diff
@@ -157,13 +157,21 @@ def printed_badges_staff(self, out, session):
 
         # part 1, include only staff badges that have an assigned name
         uber.reports.PersonalizedBadgeReport().run(out, session,
-            sa.Attendee.badge_type == c.STAFF_BADGE,
-            sa.Attendee.badge_num != None,
+            Attendee.badge_type == c.STAFF_BADGE,
+            Attendee.badge_num != None,
             order_by='badge_num')
 
         # part 2, include a bunch of extra badges so we have some printed
```

Comment:
I don't think we need both this and part 3 - they both accomplish the same thing. At the very least, maybe bump this down to 5 or something, for the much rarer case of someone being totally forgotten (vs the less-rare case of a department head not getting all their staff badges in on time)